py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -9,6 +9,9 @@
 
 from py_gtfs_rt_ingestion import batch_files, file_list_from_s3
 
+import logging
+logging.basicConfig(level=logging.INFO)
+
```

Comment:
this ensures we log all info events in addition to warnings and errors.

this will log everything to stderr by default, but we can change it to stdout if we want (i think ian said thats what they do for splunk monitoring).

we can also add formatting, and change the level via input arguments if we want.